plugin/ui/completion.py

```diff
@@ -370,19 +364,17 @@ def show(self) -> None:
         completion = self.completion_manager.current_completion
         assert completion
 
-        head, *tail = completion["displayText"].split("\n")
+        first_line, *rest_lines = completion["displayText"].splitlines()
 
+        assert self._phantom_set
         self._phantom_set.update(
             [
-                self._build_phantom(head, completion["point"] + 1, completion["point"]),
-                # even if the tail is empty it is required to add an empty phantom to prevent the cursor from jumping
-                self._build_phantom(tail or "", completion["point"], inline=False),
+                self._build_phantom([first_line], sublime.Region(completion["point"] + 1, completion["point"])),
```

Comment:
I personally liked the previous implementation with points more, mostly because it was shorter

Reply:
I will revise that part back later.